legion/docs/source/legionctl.md

```diff
@@ -32,4 +32,11 @@ Another option save the token in config file. Further you can use legionctl comm
 ```bash
 legionctl login --edi <edi-url> --token <your-token>
 legionctl inspect
-```
\ No newline at end of file
+```
+
+##Model
+
+To get json web model token for a model, you can use a command:
+```bash
+legionctl generate-token --edi <edi-url> --token <your-token> --model-id <model-id> --model-version <model-version>
```

Comment:
Remove `--token <your-token>`. This parameter is optional, and it can confuse here.